src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+
+  &__star {
+    background: url("../images/star.svg") no-repeat center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  background: url("../images/star-active.svg") no-repeat center;
+}
```

Comment:
Need to fix it
<img width="353" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/5f88c245-b92b-434f-8717-96405c146178">